src/components/Editor/index.js

```diff
@@ -272,15 +274,25 @@ const Editor = React.createClass({
       selectedToken.classList.remove("selected-token");
     }
 
+    const lineOffset = 1;
+    const { left, top } = token.getBoundingClientRect();
+    const { line, ch } = ctx.cm.coordsChar({ left, top });
+    const loc = { line: line + lineOffset, column: ch };
```

Comment:
lets move this to a helper in `utils/editor/index`

perhaps: `getLocationFromToken`